xtable-api/src/main/java/org/apache/xtable/conversion/ExternalCatalog.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+package org.apache.xtable.conversion;
+
+import java.util.Map;
+
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+import org.apache.xtable.model.catalog.CatalogType;
+
+/**
+ * This class represents the configuration for an external catalog. It holds information required to
```

Comment:
Should we add some clarification that this is different than the Iceberg catalog you can configure in XTable currently?

Reply:
done.

Reply:
@vinishjail97 I think the javadoc is still too subtle here. I can't even distinguish the difference currently by reading it.

Reply:
Yeah, will refactor some stuff to keep it the style of SourceCatalog -> Multiple Target Catalogs.